front/src/routes/ApplicantsPage.svelte

```diff
@@ -80,6 +81,39 @@
     }));
     loading = false;
   });
+
+  let selectedCommittee = null;
+
+  const passesCommitteeFilters = (
+    rows: any[],
+    selectedCommittee: CommitteeType
+  ) => {
+    if (selectedCommittee == null) {
+      return rows;
+    }
+    return rows.filter(function (row) {
+      if (row.status == ApplicationStatus.ACCEPTED) {
+        return row.committee_accepted == selectedCommittee;
+      }
+      let commExists = false;
+      row.committees.forEach(function (commObj: { committee: CommitteeType }) {
+        if (commObj.committee == selectedCommittee) {
+          commExists = true;
+        }
+      });
+      return commExists;
+    });
+  };
+
+  const updateCommitteeSelection = (committee: CommitteeType) => {
+    selectedCommittee = committee;
+    return true;
+  };
+
```

Comment:
in JS you don't have to return anything in the end of function, so this could be simplified to
```suggestion
  const updateCommitteeSelection = (committee: CommitteeType) => {
    selectedCommittee = committee;
  };

```